cal/cal/models.py

```diff
@@ -583,20 +584,38 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
+    def get_hours_per_week(self, timezone, calendar_ids=None, start=None, end=None):
         """
         Returns a list of week-hour tuples corresponding to the events in this ColorCategory.
         Each week starts at the start time.
         """
         week_hours = []
         events = self.query().order_by('start')
         i = 0
-        start = events[0].start
+        # Convert start to local time
+        start = handle_time_string(str(events[0].start), timezone)
```

Comment:
You're not actually using the `start` that was passed into this function as a parameter; you're overriding it and going by the first event that is returned in the query, though you're probably already aware of this. This is best for cumulative queries but breaks if we want to zoom and look at subsets of the overall query (ex. I want to look at only April from this year)